recipes/rmdir/src/workflow.ts

```diff
@@ -0,0 +1,75 @@
+import type { SgRoot, Edit } from "@ast-grep/napi";
+
+/**
+ * Transform function that converts deprecated fs.rmdir calls
+ * with recursive: true option to the new fs.rm API.
+ *
+ * Handles:
+ * 1. fs.rmdir(path, { recursive: true }, callback) -> fs.rm(path, { recursive: true, force: true }, callback)
+ * 2. fs.rmdir(path, { recursive: true }) -> fs.rm(path, { recursive: true, force: true })
+ * 3. fs.rmdirSync(path, { recursive: true }) -> fs.rmSync(path, { recursive: true, force: true })
+ * 4. fs.promises.rmdir(path, { recursive: true }) -> fs.promises.rm(path, { recursive: true, force: true })
+ */
+export default function transform(root: SgRoot): string | null {
+  const rootNode = root.root();
+  let hasChanges = false;
+  const edits: Edit[] = [];
+
+  // Find all fs.rmdir, fs.rmdirSync, and fs.promises.rmdir calls
+  const rmdirCalls = rootNode.findAll({
+    rule: {
+      any: [
+        { pattern: "fs.rmdir($PATH, $OPTIONS, $CALLBACK)" },
+        { pattern: "fs.rmdir($PATH, $OPTIONS)" },
+        { pattern: "fs.rmdirSync($PATH, $OPTIONS)" },
+        { pattern: "fs.promises.rmdir($PATH, $OPTIONS)" }
+      ]
+    }
+  });
+
+  for (const call of rmdirCalls) {
+    const callText = call.text();
+
+    // Check if this call has recursive: true option
+    const optionsMatch = call.getMatch("OPTIONS");
+    if (!optionsMatch) continue;
+
+    const optionsText = optionsMatch.text();
+    if (!optionsText.includes("recursive") || !optionsText.includes("true")) {
+      continue;
+    }
```

Comment:
What about all the other options?

Reply:
idk if there are an impact on the codemod